docker/dataset_eval/consul-template-dataset-eval.conf

```diff
@@ -0,0 +1,11 @@
+template {
+    source = "/code/consul_config.py.ctmpl"
+    destination = "/code/config.py"
+}
+exec {
+    command = "cd /code && exec chpst -uacousticbrainz:acousticbrainz python worker_manage.py dataset_evaluator"
```

Comment:
this won't work, because consul-template doesn't run this in a subshell, it just exec's the command. We also need to make an update to handle this in Listenbrainz. My plan is that we should have a shellscript for running python commands, e.g. `start-ab-command.sh`

```
#!/bin/bash
cd /code
exec chpst -uacousticbrainz:acousticbrainz python "$@"
```
and set `command = "start-ab-command.sh python worker_manage.py dataset_evaluator"`